FileIO/CsvInterface.h

```diff
@@ -0,0 +1,183 @@
+/**
+ * @file CsvInterface.h
+ * @author Karsten Rink
+ * @date 2015-03-25
+ * @brief Definition of the CsvInterface class.
+ *
+ * @copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef CSVINTERFACE_H_
+#define CSVINTERFACE_H_
+
+#include <array>
+#include <fstream>
+#include <limits>
+#include <list>
+#include <string>
+#include <vector>
+
+// ThirdParty/logog
+#include "logog/include/logog.hpp"
+
+#include "StringTools.h"
+
+namespace GeoLib {
+	class Point;
+}
+
+namespace FileIO {
+
+/**
+ * Interface for reading CSV file formats.
+ */
+class CsvInterface {
+
+public:
+	/** 
+	 * Reads 3D points from a CSV file. It is assumed that the file has a header
+	 * specifying a name for each of the columns. The first three columns will be
+	 * interpreted as x-, y- and z-coordinate, respectively.
+	 * \param fname    Name of the file to be read
+	 * \param delim    Deliminator, default is ','
+	 * \param points   A vector containing the 3D points read from the file
+	 * \return An error code (0 = ok, 0<i<max = number of skipped lines, -1 error reading file)
+	 */
+	static int readPointsFromCSV(std::string const& fname, char delim,
```

Comment:
I would remove "FromCSV" from the function name, because it is obvious from the class name